DMTR-121.tex

```diff
@@ -131,6 +187,176 @@ \section{Detailed Test Results}
 
   \subsection{Test Cycle LVV-C56 }
 
+Open test cycle {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testrun/LVV-C56}{LDM-503-8 Enable spectragraph data viewable by LSP.
+}} in Jira.
+
+  LDM-503-8 Enable spectragraph data viewable by LSP.
+\\
+  Status: Not Executed
+
+  Image data needs to be created and available for transfer to NCSA. ~The
+data will be transferred to NCSA and made available to scientists for
+viewing and verification.~ ~~
+
+
+  \subsubsection{Software Version/Baseline}
+    Not provided.
+
+  \subsubsection{Configuration}
+    ATS system connected to spectragraph DAQ is required to have well formed
+image data. ~Well formed means ``good'' image and correct headers.
+~\\[2\baselineskip]
+
+
+  \subsubsection{Test Cases in LVV-C56 Test Cycle}
+
+
+    \paragraph{Test Case LVV-T454 }\mbox{}\\
+
+Open  \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T454}{\textit{ LVV-T454 } }
+test case in Jira.
+
+    \begin{itemize}
+\tightlist
+\item
+  Acquire spectragraph image data, transfer that data to NCSA, ingest
+  data into some Butler (G2 or G3 when available), and enable viewing of
+  data on LSP. ~
+\end{itemize}
+
+
+    {\bf Preconditions}:\\
+    Data must be well formed on Spectragraph data archiving system (ATS).
+~Well-formed means ``good image'' and correct headers. (\citeds{LSE-400}) ~
```

Comment:
\citeds{LSE-400} reders as (?) on https://dmtr-121.lsst.io/v/DM-16455/index.html 
Has this reference been correctly included?